fala/apps/adviser/forms.py

```diff
@@ -69,67 +70,88 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault("label_suffix", "")
         super(AdviserRootForm, self).__init__(*args, **kwargs)
 
-    def clean(self):
+    def clean(self):  # noqa: max-complexity=13 (increasing the complexity of a function so flake8 is happy)
```

Comment:
this method `def clean` is handling a lot of logic, hence why Flake8 is also complaining. Rather than ignoring the Flake8 complaint, it would be better to try and split the logic out into small methods to make this more readable/so the method has one responsibility only

Reply:
Yes - now broken up a bit and no longer need this 🙂